ghedesigner/utilities.py

```diff
@@ -84,3 +88,132 @@ def solve_root(x, objective_function, lower=None, upper=None, abs_tol=1.0e-6, re
         x = upper
 
     return x
+
+
+def write_idf_object(data: list):
```

Comment:
New functions to write IDF objects and the IDF file.

Reply:
EpJSON could be useful at some point. This is obviously only a partial IDF, so I'm just writing the relevant IDF objects currently.